### PR TITLE
Add required algorithm attribute to ssh sign call

### PR DESCRIPTION
**Pull request by trebortech**
this commit is related to issue #32

### CODE REVIEW

File: yubihsm/objects.py [hunk 1]
```diff
@@ -733,9 +733,9 @@ def sign_ssh_certificate(
 
         :param template_id: The ID of the SSH TEMPLATE to use.
         :param request: The SSH certificate request.
-        :return: The signed SSH certificate.
+        :return: The SSH certificate signature.
         """
-        msg = struct.pack("!HH", self.id, template_id) + request
+        msg = struct.pack("!HHB", self.id, template_id, algorithm.value) + request
```
Comment (dainnilsson) [inline yubihsm/objects.py:738]: Please change `algorithm.value` to just `algorithm` here, the rest isn't needed.

**Comment by dainnilsson** [comment]
Sorry for the delay, this has now been merged into master. Note however that the code on master is not backwards compatible with python-yubihsm 2.x, and will eventually be released as a new major version (python-yubihsm 3.0). Unfortunately that likely won't happen until next year sometime.

If you require a released version please let me know, and I'll see if we can prioritize doing a maintenance release of the 2.x branch and include this fix sooner.

**Comment by peyr0l** [comment]
Thank you for merging the fix.

If backporting this to 2.x line wouldn't take much time (I believe it should not ;-)) I would appreciate it much. We're trying to use this ceritfication functionality in a tool in our organization and surely prefer to use a stable, released version of the code.

**Comment by dainnilsson** [comment]
This has now been released as version 2.1.2.

**Comment by peyr0l** [comment]
Thanks for the backport! :)

**Pull request merged**